client/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_KEYCLOAK_REALM=${KEYCLOAK_REALM}
```

Comment:
If the env vars on the right (`KEYCLOAK_REALM`, `KEYCLOAK_AUTH_URL`, `KEYCLOAK_FE_CLIENTID`) are already required, why don't we just use those in `keycloak.js`? As far as I can tell, this file is simply mapping the names of env vars, which could be completely avoided.

Reply:
Env vars on the client side must start with `REACT_APP_` according to the [docs](https://create-react-app.dev/docs/adding-custom-environment-variables/) unless you eject the app, so if we want to share the same var between `server` and `client` we have to map to that prefix first.